tests/it/plugin/startPlugin.test.js

```diff
@@ -2,15 +2,52 @@
 const {describe, it} = require('mocha');
 const {expect} = require('chai');
 const fetch = require('node-fetch');
-const startPlugin = require('../../../src/plugin/startPlugin');
+let startPlugin = require('../../../src/plugin/startPlugin');
 
 describe('start plugin', () => {
+  let getCloseServer, __module;
+
+  beforeEach(() => {
+    __module = {exports: () => {}};
+    getCloseServer = startPlugin(__module);
+  });
+
+  afterEach(async () => {
+    __module = null;
+    await getCloseServer()();
+  });
+
   it('starts plugin server and patches module exports', async () => {
-    const __module = {exports: () => {}};
-    const getCloseServer = startPlugin(__module);
     const {eyesPort} = await __module.exports();
     const resp = await fetch(`http://localhost:${eyesPort}/hb`);
     expect(resp.status).to.equal(200);
-    await getCloseServer()();
+  });
+
+  it('patches module exports with enabled eyes pref', async () => {
+    const {eyesIsDisabled} = await __module.exports();
+    expect(eyesIsDisabled).to.be.false;
+  });
+
+  describe('with eyes disabled', () => {
+    before(() => {
+      process.env['APPLITOOLS_IS_DISABLED'] = true;
+      delete require.cache[require.resolve('../../../src/plugin/startPlugin')];
+      startPlugin = require('../../../src/plugin/startPlugin');
+    });
+
+    beforeEach(() => {
+      __module = {exports: () => {}};
+      getCloseServer = startPlugin(__module);
+    });
+
+    after(() => {
+      delete process.env['APPLITOOLS_IS_DISABLED'];
+      delete require.cache[require.resolve('../../../src/plugin/startPlugin')];
+    });
+
+    it('patches module exports with disabled eyes pref', async () => {
+      const {eyesIsDisabled} = await __module.exports();
+      expect(eyesIsDisabled).to.be.true;
+    });
```

Comment:
I have a feeling this could be less verbose if `__module` was moved into every `it`. I also don't like nested describes, but that's my personal taste. This code is still straight forward so I find it easy to understand what you are doing here.